tests/lora/test_lora_layers_sd3.py

```diff
@@ -287,3 +287,19 @@ def test_simple_inference_with_transformer_fuse_unfuse(self):
         self.assertTrue(
             np.allclose(ouput_fused, output_unfused_lora, atol=1e-3, rtol=1e-3), "Fused lora should change the output"
         )
+
+    def test_sd3_lora(self):
+        components = self.get_dummy_components()
+
+        pipe = self.pipeline_class(**components)
+        pipe = pipe.to(torch_device)
+        pipe.set_progress_bar_config(disable=None)
+
+        lora_model_id = "OzzyGT/lora_test"
+        lora_filename = "lora_diffusers_format.safetensors"
+        pipe.load_lora_weights(lora_model_id, weight_name=lora_filename)
+        pipe.unload_lora_weights()
+
+        lora_model_id = "OzzyGT/lora_test"
+        lora_filename = "lora_peft_format.safetensors"
+        pipe.load_lora_weights(lora_model_id, weight_name=lora_filename)
```

Comment:
Thanks for adding the tests. 

We should perhaps make this a SLOW one or require this test to have a GPU (`@require_torch_gpu`). And also add a comment to explain why we had to add this test. I think adding a mention of this PR is just fine.